src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
well, it does alot of difference, come online, and try to find the difference of /rainbownick and /nicknyan